src/app/core/http-services/managed-groups.service.ts

```diff
@@ -1,29 +1,29 @@
 import { Injectable } from '@angular/core';
```

Comment:
I think there's a bug [here (group composition user list)](https://dev.algorea.org/branch/feature/convert-all-service-call-to-decoder/en/#/groups/by-id/672913018859223173;path=52767158366271444/details/members)

Reply:
Also need to add Session type to doc https://france-ioi.github.io/algorea-devdoc/api/#operation/invitationsView

Reply:
> Please, create BE issue for doc https://france-ioi.github.io/algorea-devdoc/api/#operation/groupUserRequestsView
Currently API returns grade as nullable

I did not understand this comment, there's no `grade` property in the response schema, is it `group` ?
And it appears required but is actually nullable ?

Reply:
In doc - user.grade described as optional, type number. But actual API returned null value. So I can assume or API returns wrong value or doc should be correct like user.grade could be Nullable.